src/main/webapp/js/common.es6

```diff
@@ -988,7 +988,7 @@ function setChevronToDown(chevronContainer) {
 function toggleChevron(clickedElement) {
     const $clickedElement = $(clickedElement);
     const isChevronDown = $clickedElement.find('.glyphicon-chevron-down').length > 0;
-    const $chevronContainer = $clickedElement.find('.glyphicon');
+    const $chevronContainer = $clickedElement.find('.glyphicon-chevron-up, .glyphicon-chevron-down');
```

Comment:
It is indeed a concern, however I do not see the direct reason why this should be included in this PR. However, since it is a small change, I'll let it pass.